packages/node/src/indexer/api.service.test.ts

```diff
@@ -61,38 +80,80 @@ describe.skip('ApiService', () => {
     app = module.createNestApplication();
     await app.init();
     apiService = app.get(ApiService);
+    if (kyve) {
+      (apiService as any).nodeConfig._config.kyveEndpoint =
+        'https://api-us-1.kyve.network';
+      (apiService as any).nodeConfig._config.kyveStorageUrl =
+        'https://arweave.net';
+    }
     await apiService.init();
   };
 
-  beforeAll(async () => {
-    await prepareApiService();
+  describe('KYVE Api service', () => {
```

Comment:
These tests don't seem relevant here. I think if anything there should be a test to check that kyve is used to fetch blocks and that it falls back if it fails or is not used.

These tests currently just test kyve functionality with all the baggage of api service